.github/workflows/license_tests.yml

```diff
@@ -1,11 +1,6 @@
 name: Run License Tests
 on:
   push:
-    branches:
```

Comment:
Is this change intentional? This will make test run on any `push` event

Reply:
its a copy paste from another phal plugin